helm/config/backend.yaml

```diff
@@ -7,6 +7,9 @@ docker:
 
 k8s:
   namespace: {{ .Values.backend.k8sSessionNamespace }}
+  {{- if eq .Values.target "azure" }}
+  ingressClassName: nginx
+  {{- end }}
```

Comment:
I'd rather just take it from the `values.yaml` and overwrite it there. A good place would be `cluster.namespaces.sessions.ingressClassName`.

The ingressClassName is a flexible name / string. It can be named nginx (as it is in our case), but can have any other name. To support more Kubernetes instances, let's have it configurable.

Reply:
I did not want to change Helm's `values.yml`. However putting it in the `sessions` section _does_ make sense :sweat_smile: .

Reply:
In this case, it would make sense. And when it's set to `null` per default, it doesn't have any effect on existing deployments.